src/main/java/com/samcomo/dbz/report/model/constants/ReportStatus.java

```diff
@@ -0,0 +1,14 @@
+package com.samcomo.dbz.report.model.constants;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public enum ReportStatus {
+  PUBLISHED("게시됨"),
+  DELETED("삭제됨"),
+  FOUND("찾음");
```

Comment:
```suggestion
  IN_PROGRESS("진행"),
  DELETED("종료"),
  COMPLETED("완료");
```
프론트에서 `진행중`, `완료됨` 으로 필터링되는 것을 고려해서 위처럼 네이밍하는 건 어떤가요?

Reply:
<img width="234" alt="image" src="https://github.com/SamCoMo/DBZ-Backend/assets/83532821/cba74c8d-ceed-4b57-9e2e-b2ca6e0afe89">

네네 그러네요. 진행중 , 완료, 삭제됨 으로 변경하겠습니다.